packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.styles.ts

```diff
@@ -14,15 +15,54 @@ export const getStyles = (
 
   const { palette, semanticColors } = theme;
 
+  const AboveAndBelowStyles: IStyle = {
+    position: 'absolute',
+    pointerEvents: 'auto',
+    width: '100%',
+    zIndex: 1
+  };
+
   return ({
     root: [
       'ms-ScrollablePane',
       {
-        // Insert css properties
+        overflowY: 'auto',
+        maxHeight: 'inherit',
+        height: 'inherit',
+        // @todo this doesn't work with JS
```

Comment:
Is this a regression if we merge as-is?

Reply:
Actually I should ask @dzearing about this.  Unfortunately it is not included in the JS Api, so I it is not as simple as adding an IRawStyle.  I did a little research, but the general comment was that if you want to get this specific with your styles, don't style with JS as it's just not there yet.  This will not cause a visual regression though, only cause scroll behavior to differ on specific webkit browsers like Opera and Chrome.

Also, @betrue-final-final do you know how important this rule is design wise, or if it was just added out of convenience?